AzureCalling/Controllers/CallViewController.swift

```diff
@@ -262,6 +262,7 @@ class CallViewController: UIViewController, UICollectionViewDelegate, UICollecti
     @IBAction func onToggleVideo(_ sender: UIButton) {
         sender.isSelected.toggle()
         if sender.isSelected {
+            localParticipantView.dispose()
```

Comment:
Shouldn't we dispose it only after we get a success call back from StopLocalVideoStream?

Reply:
Tried that, it's going to have the flickering behavior as stated [here](https://teams.microsoft.com/l/message/19:611ad8e0660848af8f3c80ee02f49ea6@thread.tacv2/1626824687747?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=8fac7e34-7319-4e89-95a1-c97cfe6a7344&parentMessageId=1626731965985&teamName=ACS%20Mobile%20UI&channelName=Calling%20Sample%20App&createdTime=1626824687747)

Reply:
Gave some thought about it, made sense to only dispose after StopLocalVideoStream succeed, don't want any issue if we dispose the view and then encountered an error when StopLocalVideoStream failed.

Indicator flickering might not be a big issue, probably can ask the SDK team about it.